.github/workflows/generate_package_wiki.yml

```diff
@@ -0,0 +1,102 @@
+name: Generate Package Wiki
+
+on:
+  schedule:
+    # Runs daily at a weird time (02:50 UTC) to avoid delays during periods of
+    # high loads of GitHub Actions workflow runs.
+    - cron: '50 2 * * *'  
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  generate-wiki:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Checkout Wiki Repo
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository }}.wiki
+          path: wiki
+          token: ${{ secrets.REPO_TOKEN }}
+          fetch-depth: 0
+
+      - name: Generate Wiki Content
+        id: generate-content
+        shell: pwsh
+        run: |
+          $source = "https://www.myget.org/F/vm-packages/api/v2" 
+
+          $packagesDir = "packages"
+          $packages = Get-ChildItem -Path $packagesDir -Directory | ForEach-Object { $_.Name } | Sort-Object
+
+          $wikiContent = "# Package List`n`n"
```

Comment:
I think we should remove the `Package List` header, as the page has a header already and this creates a sublevel also in the index:
# Page
![image](https://github.com/user-attachments/assets/0125f464-a5f2-4813-a35f-b3108526e84d)
# Index
![image](https://github.com/user-attachments/assets/90617d8d-b5e5-4842-bb81-0e5acdcff8d5)